packages/frontend-passthrough-headers/jest.config.ts

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  collectCoverage: true,
+  preset: "ts-jest",
+};
```

Comment:
Is it possible to move these rules to the root level? They seem like one's we'd want to apply to all the packages

As it's a `ts` file, is `module.exports` the right syntax?